Lib/test/test_types.py

```diff
@@ -421,9 +421,6 @@ def test_int__format__locale(self):
             self.assertEqual(len(format(0, lfmt)), len(format(x, lfmt)))
             self.assertEqual(len(format(0, cfmt)), len(format(x, cfmt)))
     
-    if sys.platform != "darwin":
-        test_int__format__locale = unittest.expectedFailure(test_int__format__locale)
```

Comment:
Reverting this will fix the failing test

Reply:
Oh my.. That code accidentally got included in the commit. I've fixed it!